examples/notebooks/intro_model_definition_methods.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
I love this idea of adding intro notebooks!

Reply:
Yeah, I think the notebook format actually works especially well for this kind of how-to style!

Reply:
Agreed!